docs/articles/documentation/using-testcafe/configuration-file.md

```diff
@@ -0,0 +1,715 @@
+---
+layout: docs
+title: Configuration File
+permalink: /documentation/using-testcafe/configuration-file.html
+---
+# Configuration File
+
+TestCafe uses the `.testcaferc.json` configuration file to store its settings.
+
+> Important! Settings you specify when you run TestCafe from the [command line](command-line-interface.md) and [programming interfaces](programming-interface/README.md) override settings from `.testcaferc.json`. TestCafe prints a message in the console when this happens.
+
```

Comment:
I think we need an example file with all defined options or smth like that.

Reply:
Good idea, I was thinking about it, I just don't know how to test this file to ensure all options together are valid and non-contradictory. (I'm not sure I could possibly run TestCafe on my computer with all the possible options set).

Anyway, I'll add a file to `/examples`.